indico/modules/events/registration/templates/management/regform_modify.html

```diff
@@ -7,6 +7,7 @@
 {% endblock %}
 
 {% block content %}
+    {{ template_hook('before-regform', event=event, regform=regform, management=management, registration=none) }}
```

Comment:
are you sure this is needed here? it's for editing the regform itself, not for using it for a registration

Reply:
I'm sure. It's useful if the conference manager wants to customize the block above from the UI.
In my case, this block looks like a section of the registration form, and the manager can consistently customize title and subtitle as per the other section in the registration form.
The manager should perceive it as a "_regular_" section

Reply:
I wonder if it should be a different signal. If not, this signal **needs** a new kwarg (`regform_management`? sounds awful but i can't think of a better one right now) that is usually set to false and true here.

Reply:
I get your concerns, but consistently with the show up of _angular_ registration-form, It sounds to me more like `above-regform` and `below-regform` for two pieces of bread that you graphically need to create your sandwich, with the registration form in between.

Reply:
yes sure, i just meant something like `above-regform-management` because i think it's unlikely that one wants to inject the same code on the two pages

Reply:
I consider these as an _extension_ of the _angular_ registration form.

Reply:
So, what's the purpose of the `management` parameter?
you can always implement a function like:
```
if management:
 do_this()
else:
 do_that()
return render_template(same_template)
```
In my case, the only difference is just a different parameter passed to my javascript function.
Everything else is exactly the same.

Reply:
But the current `management` parameter indicates if the registration is being edited  in the management area or not. 

Now that you are adding a hook to `regform_modify.html`, this will run when managing the **registration form itself**. And a plugin that wants to show something during registration, most likely does not want to show this when the registration form is being edited.

Reply:
And the reason why I suggested a separate signal:

Right now the cern_access plugin uses this code to inject its fields when someone registers for the event:

```python
    def _get_personal_data_form(self, event, regform, management, registration=None, **kwargs):
if management or registration is not None:
    return
```

If you start calling the hooks when managing the regform, this code will also run. Of course it's no problem for us to update the plugin - but IMO those are two totally different things (registrations vs registration forms), so separate hooks would avoid this problem altogether.

Reply:
Ok, but at this point, both of them should be separated. How can we proceed? you separate yours while I separate mine?
Do you separate both?

Reply:
not sure what you mean... the existing hooks only run when you create/edit registrations. so i'd add the before-regform hooks in those files where we already have `after-regform` ones.

and if you do need hooks in regform management as well, then (maybe as a separate commit) add new before/after signals. `<before|after>-regform-setup` or something like that

Reply:
OK, let's proceed this way. I need those hooks for the regform management. Later on, I'll create the `<before|after>-regform-setup`.